tensorflow/contrib/ffmpeg/default/ffmpeg_lib.cc

```diff
@@ -69,7 +69,10 @@ bool IsBinaryInstalled(const string& binary_name) {
   for (const string& dir : str_util::Split(path, ':')) {
     const string binary_path = io::JoinPath(dir, binary_name);
     char absolute_path[PATH_MAX + 1];
-    ::realpath(binary_path.c_str(), absolute_path);
+    if (::realpath(binary_path.c_str(), absolute_path)) {
```

Comment:
should this be comparing to nullptr?  It seems here that we will reject valid paths.